rtl/eclrtl/rtlformat.cpp

```diff
@@ -934,8 +934,7 @@ void CPropertyTreeWriter::outputBeginDataset(const char *dsname, bool nestChildr
         return;
     StringBuffer dsNameUtf8;
     outputXmlUtf8(rtlUtf8Length(strlen(dsname), dsname), dsname, nullptr, dsNameUtf8);
-    target = target->addPropTree(dsNameUtf8);
-    nestedLevels.append(*target);
+    target->addProp("name", dsNameUtf8);
```

Comment:
This change doesn't look correct - the end dataset still pops a value from nestedLevels.

Reply:
@jakesmith should this be @name?
(comparing with CommonXmlWriter::outputBeginDataset)

Reply:
Hm, I'm pretty sure it should be _and_ was, as I tested the output was <Dataset name='blah'>
I must have reverted it during testing.

I'll double check and change.